source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ACRContentStackView: UIView
```

Comment:
By having embedding UIStackView inside UIView, it gives more flexibility and control of its look. For example, if a UIStackView is needed to get bigger than its intrinsic size when it is rendered, it gives better result if the embedding view increases its size because it gives  the embedded UIStackView options to increase its size also or remain same size and add more padding around it maintaining its look.

---
In reply to: [128642527](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128642527) [](ancestors = 128642527,128332929,128110253)